### PR TITLE
Checkbox in Graph Setup to show/hide grid

### LINKED ISSUE #407: Grid feature is hard to use
To display a grid you need to know that you should hold alt and click a field in the legend. Then you also need to turn "master grid switch" on (G key).

### PR DESCRIPTION
**Pull request by wolkesson**
Suggestion to fix #407

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1269,7 +1271,7 @@ html:not(.has-config) #status-bar .configuration-file-name {
 
 ***/
 
-input[type="checkbox"] { 
+input[type="checkbox"].ios-switch {
```
Comment (mikeller) [inline css/main.css]: What's the `.ios-switch` for?

Comment (wolkesson) [reply css/main.css]: All checkboxes are rendered as ios-switches, but in this table I had too limited space. So I wanted a checkbox instead. Also an switch was a bit confusing. "Am I turning something on? Why is my field turned off by default?"

Comment (mikeller) [reply css/main.css]: Adding screenshots of proposed changes to the pull request often helps with illustrating what the change does.

File: js/graph_config_dialog.js [hunk 2]
```diff
@@ -323,7 +324,7 @@ function GraphConfigurationDialog(dialog, onSave) {
                     },
                     color: $('select.color-picker option:selected', this).val(),
                     lineWidth: parseInt($("input[name=linewidth]", this).val()),
-                    grid: (($('div#grid', this).attr("value")==="true")?true:false)
+                    grid: (($('input[name=grid]', this).is(':checked')))
```
Comment (McGiverGim) [inline js/graph_config_dialog.js]: To make Sonar happy:
- remove the useless extra parenthesis
- add a trailing comma to the last element of the list
This issues were there before your change, but now that you are modifying the line... 🤪

File: css/main.css (hunk 1 above)
Comment (wolkesson) [reply css/main.css]: image

Before this change any checkbox without ios-switch class was hidden. The ios-switch hides the checkbox and amend it's own graphics.

File: js/graph_config_dialog.js (hunk 2 above)
Comment (wolkesson) [reply js/graph_config_dialog.js]: Pushed change

**Pull request merged**